npm/builder.js

```diff
@@ -481,8 +479,8 @@ Builder.prototype.compile = function() {
 
   log.title('copy resources');
   log.debug('copy asciidoctor.css');
-  // FIXME: Find a more robust implementation to get the Asciidoctor gem install path
-  var asciidoctorGemPath = child_process.execSync("ruby -e \"print (Gem::Specification.find_by_name 'asciidoctor').full_gem_path\"");
+  // REMIND: Asciidoctor is now a Git submodule
+  var asciidoctorGemPath = 'asciidoctor';
```

Comment:
I agree that moving the extensions is a separate (discrete) effort and not something we want to wait on.

I think we could mark a tag in the repository without releasing a gem. What do you think about 0.0.1 instead or perhaps even a pre-release name, like 1.0.0.preview.1?